src/chart/components/dynamic-objects/dynamic-objects.model.ts

```diff
@@ -95,6 +101,7 @@ export class DynamicObjectsModel extends ChartBaseElement {
 		const targetPos = paneList.getNodePosition(targetNode);
 		paneList.removeAt(targetPos);
 		this.modelIdToObjectMap.delete(id);
+		this.uniqueObjects[obj.paneId].delete(obj.parentId ?? obj.id);
```

Comment:
add if check, because `this.uniqueObjects[obj.paneId]` might be undefined. It's not likely to happen, but user can call an API manually with invalid data